cpp/FileTransferAgent.hpp

```diff
@@ -27,6 +33,99 @@ class IStorageClient;
 
 class FileTransferExecutionResult;
 
+constexpr unsigned long MILLI_SECONDS_IN_SECOND = 1000;
+
+class RetryContext
```

Comment:
Rename to PutRetryContext

Reply:
Thats class, Its instance is created as putRetryCtx